### PR TITLE
Astrolabe Direct Destination Documentation

### PR DESCRIPTION
**Pull request by samihamine**
### Proposed changes

Add Documentation for Astrolabe.so.

### Merge timing

- ASAP, please.

### CODE REVIEW

File: src/connections/destinations/catalog/astrolabe/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+This destination is maintained by Astrolabe. For any issues with the destination, [contact the Astrolabe Support team](mailto:team@astrolabe.so).
+
+## Getting Started
+
+{% include content/connection-modes.md %}
+
+1. Login to your [Astrolabe workspace](https://console.astrolabe.so/login).
+2. Go to the [Connectors page](https://console.astrolabe.so/connectors) and click **Add Connector**.
+3. Select the Segment.com Connector and sign in to your Segment account to grant Astrolabe access.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/astrolabe/index.md:15]: I see you are referrencing to some type of OAuth flow here, I'm afraid we currently do not support this for new destinations. 

Are you able to provide users with an API key to input manually? If so, please update these instructions to reflect that

**Comment by samihamine** [comment]
Hi @tcgilbert ,

I've made the necessary changes to the documentation, removing references to the OAuth flow and adding instructions for users to obtain the API key manually. Please review it when you have a moment.

Thanks.

File: src/connections/destinations/catalog/astrolabe/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Astrolabe Destination
```
[the remark this answers is not in the document]
Comment (tcgilbert) [reply src/connections/destinations/catalog/astrolabe/index.md:2]: @samihamine can you commit this change as well? I cannot update due to permissions on your fork

**Comment by pwseg** [comment]
@samihamine @tcgilbert I created #5172 to deploy these docs.

**Pull request merged**